docs/quickstart.rst

```diff
@@ -588,43 +593,41 @@ For this example, we'll create a FileSourceConnector, a FileSinkConnector and di
     [2016-08-25 18:25:19,665] INFO Herder started (org.apache.kafka.connect.runtime.distributed.DistributedHerder)
     [2016-08-25 18:25:19,676] INFO Kafka Connect started (org.apache.kafka.connect.runtime.Connect)
 
-  We will now create our first connector for reading a file from disk.  To do this, let's start by creating a file with some data. Again, if you are running Docker Machine then you will need to SSH into the VM to run these commands by running ``docker-machine ssh <your machine name>``. (You may also need to run the command as root).
+  We will now create our first connector for reading a file from disk. To do this, let's start by creating a file with some data:
 
   .. sourcecode:: bash
 
-    docker-machine ssh confluent
-
-    docker@confluent:~$ seq 1000 > /tmp/quickstart/file/input.txt
+    $ docker exec kafka-connect sh -c 'sec 1000 > /tmp/quickstart/file/input.txt'
 
 Now create the connector using the Kafka Connect REST API. (Note: Make sure you have ``curl`` installed!)
 
   Set the ``CONNECT_HOST`` environment variable.  If you are running this on Docker Machine, then the hostname will need to be ``docker-machine ip <your docker machine name>``. If you are running on a cloud provider like AWS, you will either need to have port ``28082`` open or you can SSH into the VM and run the following command:
 
   .. sourcecode:: bash
 
-    docker@confluent:~$ export CONNECT_HOST=localhost
+    $ export CONNECT_HOST=localhost
 
   The next step is to create the File Source connector.
 
   .. sourcecode:: bash
 
-    docker@confluent:~$: curl -X POST \
+    $ docker exec kafka-connect curl -s -X POST \
       -H "Content-Type: application/json" \
-      --data '{"name": "quickstart-file-source", "config": {"connector.class":"org.apache.kafka.connect.file.FileStreamSourceConnector", "tasks.max":"1", "topic":"quickstart-data", "file": "/tmp/quickstart/input.txt"}}' \
+      --data '{"name": "quickstart-file-source", "config": {"connector.class":"org.apache.kafka.connect.file.FileStreamSourceConnector", "tasks.max":"1", "topic":"quickstart-data", "file": "/tmp/quickstart/file/input.txt"}}' \
       http://$CONNECT_HOST:28082/connectors
```

Comment:
Okay, I understand that we probably have to run `curl` from within the container so that everybody's environment is the same, but it sure seems like the user might want to run `curl` on their local machine. Otherwise, it's like you have to first SSH into the machine in order to use Connect's REST API. Perhaps that's for a different issue.